__tests__/__fixtures__/output/node/axios/application-form-encoded.js

```diff
@@ -1,10 +1,15 @@
-var axios = require("axios").default;
+const axios = require("axios").default;
+const { URLSearchParams } = require('url');
+const encodedParams = new URLSearchParams();
 
-var options = {
+encodedParams.set('foo', 'bar');
+encodedParams.set('hello', 'world');
+
+const options = {
   method: 'POST',
   url: 'https://httpbin.org/anything',
   headers: {'content-type': 'application/x-www-form-urlencoded'},
-  data: {foo: 'bar', hello: 'world'}
+  data: encodedParams
```

Comment:
Made this change because this data was getting to HTTPBin actually as the following:

```js
{
  args: {},
  data: '',
  files: {},
  form: { '{"foo":"bar","hello":"world"}': '' },
  headers: {
    Accept: 'application/json, text/plain, */*',
    'Content-Length': '29',
    'Content-Type': 'application/x-www-form-urlencoded',
    Host: 'httpbin.org',
    'User-Agent': 'axios/0.21.4',
    'X-Amzn-Trace-Id': 'Root=1-613fc81c-33b383d80f53e0750840173b'
  },
  json: null,
  method: 'POST',
  origin: 'REDACTED',
  url: 'https://httpbin.org/anything'
}
```

With this fix:

```
{
  args: {},
  data: '',
  files: {},
  form: { foo: 'bar', hello: 'world' },
  headers: {
    Accept: 'application/json, text/plain, */*',
    'Content-Length': '19',
    'Content-Type': 'application/x-www-form-urlencoded',
    Host: 'httpbin.org',
    'User-Agent': 'axios/0.21.4',
    'X-Amzn-Trace-Id': 'Root=1-613fc852-1b54e94f1b77ddac478d20a4'
  },
  json: null,
  method: 'POST',
  origin: 'REDACTED',
  url: 'https://httpbin.org/anything'
}
```